nautobot/core/settings.json

```diff
@@ -137,80 +167,100 @@
         },
         "CELERY_TASK_SOFT_TIME_LIMIT": {
             "type": "integer",
-            "default": 300
+            "default": 300,
+            "description": "The global Celery task soft timeout (in seconds)."
         },
         "CELERY_TASK_TIME_LIMIT": {
             "type": "integer",
-            "default": 600
+            "default": 600,
+            "description": "The global Celery task hard timeout (in seconds)."
         },
         "CELERY_WORKER_PROMETHEUS_PORTS": {
             "type": "array",
             "items": {
                 "type": "integer"
             },
-            "default": []
+            "default": [],
+            "description": "The global Celery task hard timeout (in seconds)."
         },
         "CELERY_WORKER_REDIRECT_STDOUTS": {
             "type": "boolean",
-            "default": true
+            "default": true,
+            "description": "If enabled stdout and stderr of running jobs will be redirected to the task logger."
         },
         "CELERY_WORKER_REDIRECT_STDOUTS_LEVEL": {
             "type": "string",
-            "default": "WARNING"
+            "default": "WARNING",
+            "enum": ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
+            "description": "The log level of log messages generated by redirected job stdout and stderr."
         },
         "CHANGELOG_RETENTION": {
             "type": "integer",
-            "default": 90
+            "default": 90,
+            "description": "The number of days to retain logged changes (object creations, updates, and deletions)."
         },
         "CONFIG_CONTEXT_DYNAMIC_GROUPS_ENALBED": {
             "type": "boolean",
-            "default": false
+            "default": false,
+            "description": "If True, it will be possible to apply Config Context objects to Devices and Virtual Machines via Dynamic Group membership. When set to False this behavior will not be available."
         },
         "CONTENT_TYPE_CACHE_TIMEOUT": {
             "type": "integer",
-            "default": 0
+            "default": 0,
+            "description": "The number of seconds to cache the content type accessible via a object's class property Object._content_type_cached."
         },
         "CORS_ALLOW_ALL_ORIGINS": {
             "type": "boolean",
-            "default": false
+            "default": false,
+            "description": "The number of seconds to cache the content type accessible via a object's class property Object._content_type_cached."
         },
         "CORS_ALLOWED_ORIGINS": {
             "type": "array",
             "items": {
-                "type": "string"
+                "type": "string",
+                "format": "uri"
             },
-            "default": []
+            "default": [],
+            "description": "If True, all origins will be allowed. Other settings restricting allowed origins will be ignored."
```

Comment:
Looks like this should belong to `CORS_ALLOW_ALL_ORIGINS`, not `CORS_ALLOWED_ORIGINS`?